File: mixin-base-test/src/test/java/net/questcraft/apitests/SecondMixinTestCase.java
package net.questcraft.apitests;

import net.yakclient.mixin.api.Injection;
import net.yakclient.mixin.api.InjectionType;
import net.yakclient.mixin.api.Mixer;

@Mixer("net.questcraft.apitests.MixinSourceClassTest")
public abstract class SecondMixinTestCase {
    @Injection(type = InjectionType.BEFORE_RETURN)
    public void printTheString() {
        System.out.println("I also wanna print this");
    }
}

File: mixin-base-test/src/test/kotlin/net/questcraft/apitests/SecondMixinTestCase.kt
package net.questcraft.apitests

import net.yakclient.mixin.api.Injection
import net.yakclient.mixin.api.InjectionType
import net.yakclient.mixin.api.Mixer

@Mixer("net.questcraft.apitests.MixinSourceClassTest")
abstract class SecondMixinTestCase {
    @Injection(type = InjectionType.BEFORE_RETURN)
    fun printTheString() {
        println("I also wanna print this")
    }
}